src/components/Contact/contact.css

```diff
@@ -0,0 +1,79 @@
+.contact-form {
+  width: 340px;
+  height: 610px;
+  margin: 0 auto;
+}
+
+.form-input,
+.form-textarea {
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 20px;
+  border: 1px solid transparent;
```

Comment:
something new? Why do you use 1px tranparent borders?

Reply:
fixed